src/js/BookReader.js

```diff
@@ -1584,44 +1584,61 @@ BookReader.prototype.isFullscreen = function() {
   return this.isFullscreenActive;
 };
 
-BookReader.prototype.toggleFullscreen = function() {
-  if (this.isFullscreen()) {
+/**
+ * Fullscreen mode manager
+ * - toggles fullscreen
+ * - binds keyboard controls
+ * - fires custom event
+ * @param { boolean } bindKeyboardControls
+ */
+BookReader.prototype.toggleFullscreen = function(bindKeyboardControls = true) {
+  this._fullscreenCloseHandler = function _fullscreenCloseHandler(e) {
+    if (e.keyCode === 27) this.toggleFullscreen();
+  }.bind(this);
```

Comment:
This will be re-defined every time `toggleFullscreen` is called, which would make the `unbind` always fail (because it's a new function each time). We should also be able to use arrow fns here to avoid `bind`-ing.

```suggestion
  this._fullscreenCloseHandler = this._fullscreenCloseHandler || (e) => {
    // ESC key
    if (e.keyCode === 27) this.toggleFullscreen();
  };
```